nexuse2e-core/src/main/java/org/nexuse2e/util/CertificateUtil.java

```diff
@@ -474,9 +476,9 @@ public static String getRemainingValidity(X509Certificate x509certificate) {
                     if (remMonth > 24) {
                         long year = month * 12;
                         long remYears = div / year;
-                        remaining = "(~ " + remYears + " Years remaining)";
+                        remaining = "(~ " + remYears + " years remaining)";
                     } else {
-                        remaining = "(~ " + remMonth + " Month remaining)";
+                        remaining = "(~ " + remMonth + " months remaining)";
```

Comment:
No really scope of this PR, but I think this is not correct for 1, there is one s to much => "1 months remaining"